src/shape/shape.rs

```diff
@@ -9,14 +9,13 @@ use std::sync::Arc;
 use downcast_rs::Downcast;
 
 pub trait ShapeClone<N: RealField> {
-    fn clone_box(&self) -> Box<dyn Shape<N>> {
-        unimplemented!()
-    }
```

Comment:
I'm not sure why this was here. If people really want a `Shape` that panics on clone, that probably should be something they opt into?

Reply:
Yeah, I forgot to remove this.